Library/Dungeon Fantasy/Dungeon Fantasy Adventure 1/Rivals/Mage - Wizard-Scholar.gcs

```diff
@@ -1281,8 +1281,8 @@
 					],
 					"calc": {
 						"level": 10,
-						"parry": "No",
-						"damage": "1d-2 cr"
+						"damage": "1d-2 cr",
+						"block": "9"
 					}
```

Comment:
Kick shouldn't have a block score

Reply:
I think this sheet was created with an older version of the Natural Attacks. I've deleted it and re-added.

Reply:
This was because of GCS' change to make parry/block default to 0 instead of no. Re-adding Natural Attacks fixes it because it's not mis-loading from disk.